components/heading/tests/unit/heading.spec.js

```diff
@@ -1,3 +1,43 @@
-test('TODO replace this', () => {
-	expect(true).toBe(true);
+import React from 'react';
+import { Heading } from '@westpac/heading';
+import { render } from '@testing-library/react';
+
+import { overridesTest } from '../../../../helpers/tests/overrides-test.js';
+import { nestingTest } from '../../../../helpers/tests/nesting-test.js';
+import { GEL } from '@westpac/core';
+import wbc from '@westpac/wbc';
+// import { Heading } from '@westpac/button-dropdown';
```

Comment:
If this is not going to be used, please remove it, thanks.

Reply:
Yes, shall do - look for next commit.  Line 9 came up in auto completion during coding, then I commented it out - forgot to remove it.